ad7134: Multidevice synchronization (#1349)
analogdevicesinc/hdl: docs/projects/ad7134_fmc/index.rst

Block diagram
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> The data path and clock domains are depicted in the below diagram:

Reviewer: ```suggestion
The data path and clock domains are depicted in the diagram below:
```

Author: Done.

.. image:: ad713x_hdl.svg
   :width: 800
   :align: center
   :alt: AD7134-FMC/ZED block diagram

Multidevice Synchronization
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> An important feature that is specific to this project is multidevice
> synchronization, with a single low speed ODR line, giving less than  10 ns of
> phase matching between channels on different devices, which makes it easy to
> synchronize. Applications like condition-based monitoring, power quality
> analyzer, and sonar system demand tight phase matching across high numbers of
> channels, making the digital interface design complex. To achieve tight
> synchronization, the user must configure all the  devices in slave mode and
> use the SPI to set a specific register to reset the digital interface before
> the data capture. This command must be given to all the slaves simultaneously
> using one single SPI write command. From HDL perspective, this means driving
> both CS lines at the same time and sending the same data to both ADCs. The
> CS signals are only connected when the user want to write in a specific
> register to achieve multidevice synchronization.

Reviewer: ```suggestion
An important feature that is specific to this project is multidevice
synchronization. The single low speed ODR line makes it easy to synchronize the
devices by providing less than 10 ns of phase matching between channels on
different devices.

Applications such as condition-based monitoring, power quality
analyzer, and sonar system require tight phase matching across a large numbers of
channels, making the digital interface design complex. To achieve tight
synchronization, the user must configure all the  devices in slave mode and
use the SPI to set a specific register to reset the digital interface before
the data capture. This command must be given to all the slaves simultaneously
using one single SPI write command.

From the HDL perspective, this means driving both CS lines at the same time and
sending the same data to both ADCs. The CS signals are only connected when the
user wants to write in a specific register to achieve multidevice synchronization.
```

Break into multiple paragraphs + minor rephrasing

Author: Done. Thanks!
